modules/storage-persistent-storage-pv.adoc

```diff
@@ -121,23 +121,24 @@ The following table lists the access modes:
 [cols="1,1,3",options="header"]
 |===
 |Access Mode |CLI abbreviation |Description
-|ReadWriteOnce
+|ReadWriteOnce ^[1]^
 |`RWO`
 |The volume can be mounted as read-write by a single node.
-|ReadWriteOncePod ^[1]^
+|ReadWriteOncePod ^[2]^
 |`RWOP`
 |The volume can be mounted as read-write by a single pod on a single node.
 ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
-|ReadOnlyMany
+|ReadOnlyMany ^[1]^
 |`ROX`
 |The volume can be mounted as read-only by many nodes.
-|ReadWriteMany
+|ReadWriteMany 
 |`RWX`
 |The volume can be mounted as read-write by many nodes.
 endif::[]
 |===
 --
-1. RWOP uses the SELinux mount feature. This feature is driver dependent, and enabled by default in ODF, AWS EBS, Azure Disk, GCP PD, IBM VPC Block, Cinder, and vSphere. For third-party drivers, please contact your storage vendor.
+1. RWO and RWX can use the SELinux mount feature. However, RWO/RWX SELinux Mount is a Developer Preview feature. This feature is not supported by Red Hat, and you should not enable this feature set on production or clusters that you plan to maintain over time. For more information about this feature, see section "RWO/RWX SELinux Mount".
```

Comment:
I am not sure we need to mention SELinux for RWO and RWX. I would keep only RWOP in this page for now.

Reply:
@gcharot Do you agree? No problem if you guys want this removed from here.

Reply:
@jsafrane @gcharot OK. But we still want to keep the main doc section about this feature (see https://81580--ocpdocs-pr.netlify.app/openshift-enterprise/latest/storage/understanding-persistent-storage.html#storage-persistent-storage-selinux-rwx-rwo_understanding-persistent-storage) and the RN notice, correct?

Reply:
I don't think we need to refer to it in the proper docs, RN is enough

Reply:
Closing this PR then, since RN content is in separate PR.